packages/core/src/utils/proxyUtils.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Utility to redirect remote sources via Proxy
+ *
+ * This utility is used to rewrite a request to be routed through
+ * the Proxy instead.
+ */
+
+/**
+ * Helper-Function to route remote requests through a local proxy.
+ *
+ * @example This rewrites a remote url like `https://via.placeholder.com/300.png/09f/fff` into a URI-Component-Encoded string like `/cors-proxy/https%3A%2F%2Fvia.placeholder.com%2F300.png%2F09f%2Ffff`
+ */
```

Comment:
Function descriptions should use the imperative present tense.
Also wrapping at 80 characters:
```suggestion
/**
 * Route the given url through a local proxy.
 *
 * @example
 * This rewrites a remote url like `https://via.placeholder.com/300.png/09f/fff` 
 * into a URI-Component-Encoded string like 
 * `/cors-proxy/https%3A%2F%2Fvia.placeholder.com%2F300.png%2F09f%2Ffff`
 */
```